src/stores/RenderConfigStore.ts

```diff
@@ -56,11 +56,22 @@ export class RenderConfigStore {
     static readonly GAMMA_MAX = 2;
     static readonly ALPHA_MIN = 0.1;
     static readonly ALPHA_MAX = 1000000;
+    static readonly BIAS_MIN = -1;
```

Comment:
it would be nice if we could just adjust our bias/contrast equations to a form such that all of these variables just have the same range of `[0, 1]`.

Reply:
For the 2D box, wonder if we can apply a transformation internally so that the default position of the circle is at the center of the box?

Reply:
@veggiesaurus Sure, we can modify the functions for normal bias/contrast and combine the parameters. We may need to consider that the default value for normal bias and contrast will become 0.5 and 0.5, and this is not as straightforward as 0 and 1.

Reply:
@kswang1029 not sure how we can transform the position to the center